_topic_maps/_topic_map.yml

```diff
@@ -176,7 +176,7 @@ Topics:
       File: installing-aws-localzone
     - Name: Installing a cluster with compute nodes on Wavelength Zones
       File: installing-aws-wavelength-zone
-    - Name: Installing a cluster with compute nodes on Outposts
+    - Name: Extending an AWS VPC cluster into an AWS Outpost
```

Comment:
A redirect might be needed. See the [OCP Docs Manual](https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#bookmark=id.xb255koulgqx).

Reply:
Requested in the spreadsheet 😉